packages/pds/src/app-view/services/feed/index.ts

```diff
@@ -265,20 +274,26 @@ export class FeedService {
           cur.uri,
           actorViews,
           postViews,
-          {},
+          deepEmbedViews,
         )
         acc[cur.postUri] = {
-          $type: 'app.bsky.embed.record#presented',
-          record: formatted
+          $type: 'app.bsky.embed.record#view',
+          value: formatted
             ? {
-                $type: 'app.bsky.embed.record#presentedRecord',
+                $type: 'app.bsky.embed.record#viewRecord',
                 uri: formatted.uri,
                 cid: formatted.cid,
                 author: formatted.author,
                 record: formatted.record,
+                embeds: formatted.embed
```

Comment:
this is one too many ternaries for my tired saturday brain to handle haha 🙃 

I don't quite understand the difference between undefined & the empty array here?

& similar to my prev comment, is there ever a case when the array contains more than one value?

Reply:
Just dropped a couple comments above that will hopefully make it clearer what I was going for.  However we land on it, I'll clean this up and drop some comments in the code 👍